content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -731,16 +736,27 @@ for more information.
 
 ### SecurityContextDeny {#securitycontextdeny}
 
-This admission controller will deny any pod that attempts to set certain escalating [SecurityContext](/docs/user-guide/security-context) fields. This should be enabled if a cluster doesn't utilize [pod security policies](/docs/user-guide/pod-security-policy) to restrict the set of values a security context can take.
+This admission controller will deny any pod that attempts to set certain escalating
+[SecurityContext](/docs/tasks/configure-pod-container/security-context) fields. 
```

Comment:
I'd link to the API reference here, and then add a new paragraph with a link to https://k9s.io/docs/tasks/configure-pod-container/security-context/

Fine to leave this fix as-is; better to
```suggestion
[SecurityContext](/docs/tasks/configure-pod-container/security-context/) fields.
```
though.